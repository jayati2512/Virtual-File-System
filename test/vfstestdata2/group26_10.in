createvfs vfs1 20
unmountvfs
unmountvfs vfs
mountvfs vfs2
mountvfs vfs1
mountvfs vfs1
unmountvfs vfs1
unmountvfs vfs1
