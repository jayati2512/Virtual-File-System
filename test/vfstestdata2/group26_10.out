createvfs_SUCCESS
unmountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
unmountvfs_FAILURE VFS_NOT_MOUNTED 
mountvfs_FAILURE DATA_FILE_NOT_FOUND 
mountvfs_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED 
unmountvfs_SUCCESS
unmountvfs_FAILURE VFS_NOT_MOUNTED 
